source/includes/_tipser-react-elements.md

```diff
@@ -260,4 +260,21 @@ If you'd like to unify our design with your own color-theme, you can use our pri
 If your primary color is a bright one, you might want to change also the text color of the elements with primary color background. For example in buttons by default the color we use is white.
 
 
-If you'd like to change other elements' color as well, please use our specific classes to override the styles.
\ No newline at end of file
+If you'd like to change other elements' color as well, please use our specific classes to override the styles.
+
+## Quick React example
```

Comment:
The description below will just open a plain create-react-app. Not sure how much of the value it gives to the customer 🤔 . Maybe we should recommend to copy-paste the snippet from the section "Import Tipser React Elements" to app.js file to see Tipser Elements in action?

Reply:
Good note. I removed this section altogether, I also think our docs is not a place for that.